pom.xml

```diff
@@ -29,12 +29,12 @@
 		<dependency>
 			<groupId>dev.dsf</groupId>
 			<artifactId>dsf-fhir-rest-adapter</artifactId>
-			<version>1.3.1</version>
+			<version>1.5.0</version>
 		</dependency>
 		<dependency>
 			<groupId>de.hs-heilbronn.mi</groupId>
 			<artifactId>crypto-utils</artifactId>
-			<version>3.8.0</version>
+			<version>4.0.0</version>
```

Comment:
If `mii-processes-common` stays with 3.8.0, this module should as well to be consistent.

Reply:
reset via b65ce66